brench/brench.py

```diff
@@ -87,8 +87,11 @@ def brench(config_path, files, jobs):
         config = tomlkit.loads(f.read())
 
     # Use configured file list, if none is specified via the CLI.
-    if not files and 'benchmarks' in config:
-        files = glob.glob(config['benchmarks'], recursive=True)
```

Comment:
Looks like `recursive=True` got dropped in the `glob` call.

Reply:
Done!